ReactAndroid/src/main/java/com/facebook/react/views/text/ReactTextShadowNode.java

```diff
@@ -428,6 +439,18 @@ public void setFontStyle(@Nullable String fontStyleString) {
     }
   }
 
+  @ReactProp(name = ViewProps.TEXT_DECORATION_LINE)
+  public void setTextDecorationLine(@Nullable String textDecorationLineString) {
```

Comment:
Please add a test similar to [`testFontFamilyBoldItalicStyleApplied`](https://github.com/facebook/react-native/blob/master/ReactAndroid/src/test/java/com/facebook/react/views/text/ReactTextTest.java#L264)